### DISCUSSION TITLE
sending data from one server to another

### DISCUSSION THREAD

**Discussion by rukaiyafahmida** [miguelgrinberg/python-socketio#1080] [Q&A] [answered]
I need to connect between two servers and a client. The connection will be like:    
server A will be connected and exchange data with server B at port 5000 and server B will be connected and exchange data with the client at port 8000. 

how can I write the code?

**Comment by miguelgrinberg** [comment, marked as answer]
Okay, so you have:

- Server A running on port 5000
- Server B running on port 8000
- a client connecting to B

What you need to add, is a Socket.IO client to B, so that it can connect to A. That gives you all the connections that you need, I think.
